### PR TITLE
Added registration for 'Markus Knittig'

### PR DESCRIPTION
**Pull request by mknittig**
I would like to register for JS CraftCamp.    
- [ X] My Pull Request contains a json file `$firstname_$lastname.json` or `$nickname.json`    
- [ X] The json file follows the template at https://github.com/jscraftcamp/website/blob/master/participants/_template.json and contains the mandatory fields `name`, `tags`, `when`, `what_is_my_connection_to_javascript` and `what_can_i_contribute`    
- [X ] If I can't attend, I will either send another Pull Request removing my json file or an E-Mail with the subject 'UNREGISTER' to team@jscraftcamp.org

### CODE REVIEW

**Comment by Narigo** [comment]
I believe you need to put `M-M` as T-Shirt information.

File: participants/markus_knittig.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "tags" : [ "Craftsmanship", "TDD", "NodeJS", "ES6", "arangodb", "Clean Code", "Java", "DevOps" ],
+  "when" :
+  {
+    "friday" : true,
+    "friday_party": true,
+    "saturday" : true
+  },
+  "what_is_my_connection_to_javascript" : "I'm working as a freelance developer in JavaScript projects since 2013. Worked mostly in the backend with NodeJS, but also do some frontend work with Backbone/Marionette and Angular.",
+  "what_can_i_contribute" : "Apart from general disussion about the Craftsmanship in the JavaScript World - I'm currently mostly interested in DevOps, infrastructure-as-code and microservices",
+  "tshirt" : "M",
```
Comment (choas) [inline participants/markus_knittig.json]: fit (W=women, M=Man) and size (S,M,L,XL), e.g. W-S or M-XL

**Comment by davelosert** [comment]
@mknittig : As @Narigo and @choas already commented, you need to also set the gender information for your T-Shirt Size (in your case probably `M-M`). 
If you need help adjusting the PR just let me know. As soon as you fixed that I am happy to welcome you to JSCC17 😃

**Comment by mknittig** [comment]
Fixed!

**Comment by davelosert** [comment]
Perfect. See you at JSCC17!

**Pull request merged**